### PR TITLE
Fixes #52 update advanced example

### LINKED ISSUE #52: Update advanced example to use expected file structure

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: examples/advanced_01/input/Midazolam/Midazolam.json [hunk 1]
```diff
@@ -8758,7 +8758,17 @@
       ],
       "IntestinalPermeability": [
         {
-          "Name": "optimization"
+          "Name": "optimization",
```
Comment (msevestre) [inline examples/advanced_01/input/Midazolam/Midazolam.json]: This was missing resulting in invalid snapshot

File: examples/advanced_01/input/Midazolam_Itraconazole_DDI/Midazolam_Itraconazole_DDI.json [hunk 2]
```diff
@@ -8758,7 +8758,17 @@
       ],
       "IntestinalPermeability": [
         {
-          "Name": "Fit"
+          "Name": "Fit",
```
Comment (msevestre) [inline examples/advanced_01/input/Midazolam_Itraconazole_DDI/Midazolam_Itraconazole_DDI.json]: same here

**Comment by msevestre** [comment]
@Yuri05  Some changes but all in all, managed to generate everything with the runner in 10 min!! woot woot

**Comment by Yuri05** [comment]
geil!

**Pull request merged**